framework/src/main/java/org/tron/core/net/service/adv/AdvService.java

```diff
@@ -264,13 +264,12 @@ private void consumerInvToFetch() {
     Collection<PeerConnection> peers = tronNetDelegate.getActivePeer().stream()
         .filter(peer -> peer.isIdle())
         .collect(Collectors.toList());
-
     InvSender invSender = new InvSender();
-    long now = System.currentTimeMillis();
     synchronized (this) {
       if (invToFetch.isEmpty() || peers.isEmpty()) {
         return;
       }
+      long now = System.currentTimeMillis();
       invToFetch.forEach((item, time) -> {
         if (time < now - MSG_CACHE_DURATION_IN_BLOCKS * BLOCK_PRODUCED_INTERVAL) {
```

Comment:
Shouldn't `MSG_CACHE_DURATION_IN_BLOCKS * BLOCK_PRODUCED_INTERVAL` be replaced with `TIMEOUT` here?

Reply:
+1